src/app/shared/utils/short-hash.pipe.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This pipe takes a hash (or any string) and shortens it to the first seven characters plus an ellipsis at the end.
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * This pipe is used to shorten a hash or other string to its first seven characters plus an ellipsis at the end.
+ */
+@Pipe({
+  name: 'shortHash',
+})
+export class ShortHash implements PipeTransform {
```

Comment:
I suggest to rename this to something like "Truncate" because it can be generally useful for any text output, not just for hash values.